packages/server/lib/gui/events.js

```diff
@@ -16,7 +16,8 @@ const open = require('../util/open')
 const user = require('../user')
 const errors = require('../errors')
 const Updater = require('../updater')
-const { ProjectBase } = require('../project-base')
+const ProjectStatic = require('../project_static')
```

Comment:
Much better name, still want to get away from this concept of a singleton project when we layer in GraphQL

Reply:
Yes agreed, entire open_project abstraction needs to go